tenants/ai-example/single-model-serving/base/flan-t5-small/inference-service-flan.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  annotations:
+    openshift.io/display-name: flan-t5-small
+    serving.knative.openshift.io/enablePassthrough: 'true'
+    sidecar.istio.io/inject: 'true'
+    sidecar.istio.io/rewriteAppHTTPProbers: 'true'
+  name: flan-t5-small
+  namespace: ai-example-single-model-serving
```

Comment:
I always try to move namespace up to the kustomization.yaml file when I can.  It isn't always possible but it just makes it easier to make changes to it later on.

Reply:
fixed, added to kustomize to handle the namespace